daemon/cluster/services.go

```diff
@@ -134,6 +138,35 @@ func (c *Cluster) CreateService(s types.ServiceSpec, encodedAuth string, queryRe
 
 		switch serviceSpec.Task.Runtime.(type) {
 		// handle other runtimes here
+		case *swarmapi.TaskSpec_Generic:
```

Comment:
Ok, so when I do a service update, only one node is updated. I'm not sure I see why. The update code path is just updating the service spec and sending off to swarmkit.